src/python/controller/controller.py

```
@@ -634,6 +654,25 @@ def __update_model(self):
+                            is_dir=diff.new_file.is_dir,
+                            pair_id=pc.pair_id,
+                            local_path=pc.local_path,
+                            remote_path=pc.remote_path,
+                            algorithm=self.__context.config.validate.algorithm,
+                            remote_address=self.__context.config.lftp.remote_address,
+                            remote_username=self.__context.config.lftp.remote_username,
+                            remote_password=self.__password,
+                            remote_port=self.__context.config.lftp.remote_port,
+                        )
+                        self.__validate_process.validate(req)
+                        self.logger.info("Auto-queued validation for '{}'".format(diff.new_file.name))
```

Ruff (0.15.5)

[warning] Use f-string instead of `format` call

Convert to f-string

(UP032)